README.md

```diff
@@ -28,28 +19,52 @@ You get:
  * re(1)  — command line interface for executing regular expressions
  * lx(1)  — lexer generator
 
+lx is an attempt to produce a simple, expressive, and unobtrusive
+lexer generator which is good at lexing, does just lexing,
+is language independent, and has no other features.
+
 Clone with submodules (contains required .mk files):
 
     ; git clone --recursive https://github.com/katef/libfsm.git
 
-To build and install:
+### Building
 
-    ; pmake -r install
+To build and install:
 
-To build on Linux, you may need to install `bmake`, and use it instead
-of `pmake` in the commands mentioned here.
+    ; bmake -r
+    ; bmake -r install
```

Comment:
I'm tempted to say that you're introducing the term `bmake` without explaining it here, but I'm not sure that there's a better way without making it really wordy.

Maybe add a note about `bmake`, something like:
> To build `libfsm` requires a BSD-style make, which we'll refer to as `bmake` here to clearly distinguish it from GNU make.  See the note about `bmake` below for how to choose/obtain a BSD-style make.

Reply:
I think this is overkill, but moreso it gets in the way